app/assets/stylesheets/helpers/_footer.scss

```diff
@@ -97,4 +87,33 @@
       }
     }
   }
+
+  .footer-meta {
+    .footer-meta-inner {
+      .open-government-licence {
+        .logo {
+          .open-government-licence__logo-link {
```

Comment:
I _think_ so, just because the selector from govuk_template is `#footer .footer-meta .footer-meta-inner .open-government-licence .logo a`. 

However, since we're interested in the focus state only it might be possible to trim this down a bit. If it's not possible, I'll add a `TODO` note to say that this should be improved once `govuk_template` has been removed.

Reply:
Unfortunately this selector needs to stay - but hopefully only for a bit. 

I've added a `TODO (GOVUK_TEMPLATE)` with an explanation that we can search for to find it (and probably other things) that need a refactor once govuk_template has been removed.